validator/remote/src/main/java/tech/pegasys/teku/validator/remote/apiclient/ResponseHandler.java

```diff
@@ -41,6 +45,7 @@ public ResponseHandler(final JsonProvider jsonProvider, final Class<T> responseC
     withHandler(SC_ACCEPTED, this::noValueHandler);
     withHandler(SC_NO_CONTENT, this::noValueHandler);
     withHandler(SC_SERVICE_UNAVAILABLE, this::noValueHandler);
+    withHandler(SC_BAD_GATEWAY, this::noValueHandler);
```

Comment:
This feels slightly wrong to me. If we're getting errors from the beacon chain that's not really the same as `SC_NO_CONTENT` - it should be an error that we log.  I'm not really sure why we've been mapping `SC_SERVICE_UNAVAILABLE` to just an empty response either...

Reply:
i agree that it should be logged, service unavailable and bad gateway should definitely be a logged output of some kind, and i settled on warn, but could be swayed to error...